packages/react-router-dev/vite/plugin.ts

```diff
@@ -1908,13 +1920,6 @@ async function handlePrerender(
       );
     }
   }
-
-  await prerenderManifest(
```

Comment:
Never need to prerender a manifest anymore
- `ssr: true` will handle the manifest via the server handler
- `ssr: false` will not have fog of war enabled